src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" metod="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="fsurname" class="label">
+          Surname:
+          <input
+            type="text"
+            name="fsurname"
+            id="fsurname"
+            autocomplete="off"
+          >
+        </label>
+        <label for="fname" class="label">
+          Name:
+          <input
+            type="text"
+            name="fname"
+            id="fname"
+            autocomplete="off"
+          >
+        </label>
+        <label for="fage" class="label">
+          How old are You?
+          <input
+            type="number"
+            name="fage"
+            id="fage"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label for="fbirth" class="label">
+          Full date of birth:
+          <input
+            type="date"
+            name="fbirth"
+            id="fbirth"
+          >
+        </label>
+        <label for="fterms" class="label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="fterms"
+            id="fterms"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          Registration
+        </legend>
+        <label for="femail" class="label">
+          E-mail:
+          <input
+            type="email"
+            name="femail"
+            id="femail"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label for="fpassword" class="label">
+          Password:
+          <input type="password" name="fpassword"
+            id="fpassword"
+            minlength="6"
+            maxlength="10"
+          >
```

Comment:
more than 1 attribute - from the new line
```suggestion
          <input 
            type="password" 
            name="fpassword"
            id="fpassword"
            minlength="6"
            maxlength="10"
          >
```